admiral/pkg/controller/istio/serviceentry_test.go

```diff
@@ -24,4 +27,12 @@ func TestNewServiceEntryController(t *testing.T) {
 	if serviceEntryController == nil {
 		t.Errorf("ServiceEntry controller should never be nil without an error thrown")
 	}
+
+	serviceEntry := &v1alpha32.ServiceEntry{Spec: v1alpha3.ServiceEntry{}, ObjectMeta: v1.ObjectMeta{Name: "se1", Namespace: "namespace1"}}
```

Comment:
If we're going to test added/updated/deleted, can we use the real SE handler instead of the mock one? (Same comment applies for sidecars/VS as well)

Reply:
Same as the comment above, this is to test the controller and not the handler. Those tests need to be added in the handler.